### ISSUE TITLE
Problem with Webpack ^3.12.0

### ISSUE THREAD

**Issue by radenkozec** [orchidjs/tom-select#420]
**Describe the bug**
I am using Tom-Select v 2.0.0 without problems in dev enviroment.
Great project !!!

When I tried to deploy app to production and build it using
webpack -p --env.production --env.platform=web --progress

I am getting error:
```
ERROR in app.20c21b1a64b2ce957cfc.bundle.js from UglifyJs
Unexpected token: operator (>) [app.20c21b1a64b2ce957cfc.bundle.js:50161,38]
```
**It seams like your code in not build into plain old javascript but ES6 which causes these errors?**

My webpack is  "webpack": "^3.12.0"
"babel-core": "^6.26.3",
    "babel-eslint": "7.2.3",
    "babel-loader": "^7.1.5",
    "babel-plugin-istanbul": "4.1.4",
    "babel-plugin-syntax-flow": "^6.18.0",
    "babel-plugin-transform-class-properties": "6.24.1",
    "babel-plugin-transform-decorators-legacy": "^1.3.5",
    "babel-plugin-transform-flow-strip-types": "^6.22.0",
    "babel-plugin-transform-object-rest-spread": "^6.26.0",
    "babel-polyfill": "^6.9.1",
    "babel-preset-env": "1.5.2",
    "babel-preset-es2015": "^6.13.2",
    "babel-preset-stage-1": "^6.24.1",
    "babel-register": "^6.24.0",

When I comment out part of the application where I reference Tom-Select everything works ok.
```
import TomSelect from 'tom-select';
import 'tom-select/dist/css/tom-select.default.min.css';
```

Not sure how to fix this issue?

**Comment by radenkozec** [comment]
@oyejorge Any help?

**Comment by oyejorge** [comment]
Sure seems like webpack should be able to handle ES6. Any luck with google searches?

**Comment by radenkozec** [comment]
@oyejorge All third-party libraries I used via NPM are compiled in plain old JavaScript. Is your only compiled in ES6?
**I have managed to work around this by referencing your library from CDN.**

In this project, I use older versions of WebPack, UglifyJs and Babel and I cannot update them. Due to this, I cannot use ES6-referenced libraries although I use ES6, ES7 inside the project itself.

Not sure if you would consider maybe in next versions to compile your libraries in NPM to plain old JavaScript?
